plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestShowStats.java

```diff
@@ -680,6 +681,10 @@ public void testShowStatsWithPredicate()
                         "   ('regionkey', null, 5, 0, null, 0, 4), " +
                         "   (null, null, null, null, 5, null, null)");
         assertQuery(
+                Session.builder(getSession())
+                        // Avoid producing estimate for sin(x) through filter factor
```

Comment:
let's keep it enabled and just add a single test case with `DEFAULT_FILTER_FACTOR_ENABLED` disabled (as it's non-default behavior)